### ISSUE TITLE
Add instructions for custom task configuration

### ISSUE THREAD

**Issue by 0xTim** [swift-server/vscode-swift#76]
It might be useful to repurpoe this example, just to show how users can add command line arguments to launch configurations. If not now, let's add an issue for later.

_Originally posted by @svanimpe in https://github.com/swift-server/vscode-swift/pull/74#discussion_r774712374_

**Comment by adam-fowler** [comment]
I think it is worthwhile having a more detailed documentation separate from the README that goes into more detail. This would be a good place to include things like this.

**Comment by 0xTim** [comment]
Agreed. A docs directory is probably a good thing to branch out to where we can go into lots of detail about all the options

**Comment by adam-fowler** [comment]
I think we can close this. You add command line arguments via the extension configuration. There is no need to do this via the task.json
